modules/check_in/app/services/travel_claim/claim_submission_service.rb

```diff
@@ -192,5 +207,111 @@ def raise_backend_service_exception(detail, status = 502, code = 'VA900')
         status
       )
     end
+
+    ##
+    # Extracts the last four digits of the claim number from API response
+    #
+    # @param response [Faraday::Response] the API response
+    # @return [String] last four digits of claim ID, or 'unknown' if not found
+    #
+    def extract_claim_number_last_four(response)
+      response_body = response.body.is_a?(String) ? JSON.parse(response.body) : response.body
+      claim_id = response_body.dig('data', 'claimId')
+      claim_id&.last(4) || 'unknown'
+    rescue => e
+      log_message(:error, 'Failed to extract claim number', error: e.message)
+      'unknown'
+    end
+
+    ##
+    # Sends a success notification if feature flag is enabled
+    #
+    # @param result [Hash] the successful submission result
+    #
+    def send_notification_if_enabled(result)
+      return unless notification_enabled?
+
+      template_id = success_template_id
+      claim_number_last_four = result['claimNumberLastFour'] || 'unknown'
+
+      log_message(:info, 'Sending success notification',
+                  template_id:, claim_last_four: claim_number_last_four)
+
+      CheckIn::TravelClaimNotificationJob.perform_async(
+        @uuid,
+        format_appointment_date,
+        template_id,
+        claim_number_last_four
+      )
+    end
+
+    ##
+    # Sends an error notification if feature flag is enabled
+    #
+    # @param error [Exception] the error that occurred
+    #
+    def send_error_notification_if_enabled(error)
+      return unless notification_enabled?
+
+      template_id = error_template_id
+
+      log_message(:info, 'Sending error notification',
+                  template_id:, error_class: error.class.name)
+
+      CheckIn::TravelClaimNotificationJob.perform_async(
+        @uuid,
+        format_appointment_date,
+        template_id,
+        'unknown'
+      )
+    end
```

Comment:
this is just a style concern but the send error methods look nearly identical, ideally we would have just one method that could be shared